plugin_store/database/models/announcements.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime
+from uuid import UUID
+from zoneinfo import ZoneInfo
+
+from sqlalchemy import Column, Text, Uuid
+from sqlalchemy.orm import Mapped, mapped_column
+
+from ..utils import TZDateTime, uuid7
+from .Base import Base
+
+UTC = ZoneInfo("UTC")
+
+
+def utcnow() -> datetime:
+    return datetime.now(UTC)
+
+
+class Announcement(Base):
```

Comment:
Maybe add some way to mark an announcement as inactive so we can keep old ones around?

Reply:
Wanted to reserve it for future, but maybe it's a good idea to do it now. Added.